src/flows/recommendation_api/lifehacks_flow.py

```diff
@@ -0,0 +1,64 @@
+from typing import List
+from prefect import Flow, task
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from common_tasks.corpus_candidate_set import (
+    create_corpus_candidate_set_record,
+    load_feature_record,
+    feature_group,
+    validate_corpus_items,
+)
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+FLOW_NAME = get_flow_name(__file__)
+
+CURATED_LIFEHACKS_CANDIDATE_SET_ID = "da9cb7a1-3a34-4211-b918-73819a5586c8"
+
+# Export approved corpus items by language and recency
+EXPORT_LIFEHACKS_ITEMS_SQL = """
+SELECT 
+    approved_corpus_item_external_id as "ID", 
+    topic as "TOPIC"
+FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS"
+WHERE REVIEWED_CORPUS_ITEM_UPDATED_AT >= DATEADD("day", %(MAX_AGE_DAYS)s, current_timestamp())
+AND TOPIC IN (%(CORPUS_TOPIC_LIST)s)
```

Comment:
Awesome! I always forget how parameterized lists work, and it's actually the same as strings.